examples/dotandads.amp.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html amp>
+    <head>
+        <meta charset="utf-8">
+        <title>Amp test</title>
+        <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+        <link rel='canonical' href='dotandads.amp.html'>
+        <script async src="https://cdn.ampproject.org/v0.js"></script>
+        <style>body {opacity: 0}</style><noscript><style>body {opacity: 1}</style></noscript>
+    </head>
+    <body>
+        <amp-ad width=980 height=250
```

Comment:
Sorry, I missed this before: Please merge this into `ads.amp.html`. We're trying to keep the count of examples somewhat limited.